### PR TITLE
Upgrading a couple packages

### PR DESCRIPTION
**Pull request by tkyi**
Upgrading a couple packages:
 - [github@5](https://github.com/mikedeboer/node-github/blob/master/CHANGELOG.md#500)
 - [chai-as-promised@6](https://github.com/domenic/chai-as-promised/releases/tag/v6.0.0)
 - [nock@9](https://github.com/node-nock/nock/blob/2a1bc2d68cb5e36bc4a1501b5cbe9fe3cbcd576f/CHANGELOG.md#900--2016-10-23)
 - mockery@2

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
So meta

File: package.json [hunk 1]
```diff
@@ -91,17 +91,17 @@
   },
   "devDependencies": {
     "chai": "~3.5.0",
-    "chai-as-promised": "^5.3.0",
+    "chai-as-promised": "^6.0.0",
```
Comment (petey) [inline package.json]: Is chai-as-promised actually used? There is one file that requires it, but I don't think the features of chai-as-promised are even used in it.

Comment (tkyi) [inline package.json]: Looks like we use chai-as-promised in one test for its `isRejected` function here: https://github.com/screwdriver-cd/screwdriver/blob/master/test/plugins/auth.test.js#L108

Comment (petey) [inline package.json]: fair enough.

**Comment by d2lam** [comment]
Let's not merge anything into the pipeline until it's fixed.

**Pull request merged**